time-macros/src/lib.rs

```diff
@@ -1,3 +1,5 @@
+#![no_std]
```

Comment:
I believe this is ok even when this crate is used by other projects that are not no_std.

Reply:
That's correct. I'll check the semantics on whether this is necessary, as it's only present at build time.

Reply:
Looks like it is necessary, as it's not a proc_macro crate.